Panel.c

```diff
@@ -503,5 +506,46 @@ int Panel_getCh(Panel* this) {
 #ifdef HAVE_SET_ESCDELAY
    set_escdelay(25);
 #endif
-   return getch();
+
+   int ch;
```

Comment:
Given this value is more or less directly returned everywhere anyway, what about dropping `ch` entirely and instead just return the value we assign to it directly. The only place where this is not directly obvious to do is the case without wide char support, where things would need to be done as `return last_wkey = getch();`. The upshot of this is that the compiler will actually warn if any path through this functions accidentally missed that return value. Also this would allow to flatten the code nesting quite a bit and thus make the code easier to follow IMHO.

Reply:
Incorporated this in latest. Thanks @BenBE.